apps/activitypub/src/views/notifications/notifications.tsx

```
@@ -2,19 +2,19 @@ import React, {useEffect, useRef} from 'react';
 import {ActorProperties} from '@tryghost/admin-x-framework/api/activitypub';
 import {Button, LoadingIndicator, LucideIcon, Skeleton} from '@tryghost/shade';
 
-import APAvatar from '@components/global/APAvatar';
-import Error from '@components/layout/Error';
-import FeedItemStats from '@components/feed/FeedItemStats';
-import FollowButton from '@components/global/FollowButton';
+import APAvatar from '@components/global/ap-avatar';
+import Error from '@components/layout/error';
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)